src/app/order/OrderStatus.tsx

```diff
@@ -51,12 +66,20 @@ const OrderStatus: FunctionComponent<OrderStatusProps> = ({
             />
         </p>
 
-        { order.mandateUrl && <a data-test="order-confirmation-mandate-link-text" href={ order.mandateUrl } rel="noopener noreferrer" target="_blank">
+        { showMandateAsTextOnly() ?
```

Comment:
Inline turnaries like this can be pretty hard to read, especially as it contains two conditionals.

It's better to split out the sections of JSX into their own components, e.g.

```
interface MandateTextProps { provider: string; mandate?: string; id: string; }

const MandateText = ({ provider, mandate = '', id }: MandateTextProps) => (
    <div data-test="order-confirmation-mandate-text-only">
        <br />
        <TranslatedString
            data={ { provider, mandate } }
            id={ 'order_confirmation.' + id }
        />
    </div>
);
```

Then this turnary becomes something like:

```
        { showMandateAsTextOnly || !order.mandateUrl ?
            <MandateText id={ mandateTextId } mandate={ order.mandateUrl } provider={ mandateProvider } /> :
            <MandateLink id={ mandateTextId } mandate={ order.mandateUrl } provider={ mandateProvider } /> }
```

By splitting out the defining of the JSX, we can just concentrate on what will be executed given the different scenarios.